libp2p/stream_muxer/abc.py

```diff
@@ -35,6 +34,11 @@ def __init__(
         :param peer_id: peer_id of peer the connection is to
         """
 
+    @property
+    @abstractmethod
+    def initiator(self) -> bool:
```

Comment:
suggest
```suggestion
    def is_initiator(self) -> bool:
```
Maybe we can do this renaming across the codebase all together in this PR?(`initiator` -> `is_initiator`)

Reply:
> Maybe we can do this renaming across the codebase all together in this PR?(initiator -> is_initiator)

Sounds good to me.